<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TodoListClass</class>
 <widget class="QMainWindow" name="TodoListClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TodoList</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayoutButtons">
        <item>
         <widget class="QPushButton" name="pushButtonAdd">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonDelete">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonEdit">
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="mainGridLayout">
        <item row="0" column="0">
         <widget class="QListView" name="listViewPlaned"/>
        </item>
        <item row="0" column="2">
         <widget class="QListView" name="listViewDone"/>
        </item>
        <item row="0" column="1">
         <widget class="QListView" name="listViewInProgress"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>568</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="todolist.qrc"/>
 </resources>
 <connections/>
</ui>
